.php_cs.dist

```diff
@@ -23,39 +24,54 @@ if (PHP_SAPI !== 'cli') {
 return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
+        '@DoctrineAnnotation' => true,
         '@PSR2' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'cast_spaces' => ['space' => 'none'],
+        'concat_space' => ['spacing' => 'one'],
+        'declare_equal_normalize' => ['space' => 'none'],
```

Comment:
make this
`'declare_equal_normalize' => ['space' => 'single'],`

so we've got PSR-12 compatible `declare(strict_types=1);` statements